.vscode/tasks.json

```diff
@@ -5,7 +5,6 @@
   "tasks": [
     {
       "label": "test current gem",
-      "type": "shell",
```

Comment:
Any notes on this? I don't know that I've used these but just curious

Reply:
I don't quite understand it, but this line was causing the task to use my system Ruby instead of my `asdf` Ruby. I removed it and things worked fine. 🤷‍♂️